application/views/survey/questions/answer/gender/buttons/answer.twig

```diff
@@ -13,54 +13,29 @@
 
 <!--Gender question, buttons display -->
 <!-- answer -->
-<ul class="{{ coreClass }} list-unstyled btn-group btn-group-justified" data-toggle="buttons" role="radiogroup" aria-labelledby="ls-question-text-{{basename }}">
+<div class="ls-answers answers-list gender-button btn-group btn-group-justified" role="radiogroup" aria-labelledby="ls-question-text-{{ basename }}">
```

Comment:
Why remove coreClass system ?

The reason to add it it's to try to have something fixed for SurveyTheme developper and workaround. 
Not perfect, but better than nothing.

Reply:
Hm, seems to be hard-coded in multiple places in qanda helper. Is it the `ls-answer` class you want to be constant?

Reply:
> Hm, seems to be hard-coded in multiple places in qanda helper. Is it the `ls-answer` class you want to be constant?

$coreClass yes, since QuestionTheme must use it …

Reply:
Question is where to put it. If you want it to be same everywhere, it must be defined in one place only. One suggestion is question model base class, as a property.

Reply:
Then put `ls-answer` in $coreClass, and everything else in Twig files?

Reply:
> Then put `ls-answer` in $coreClass, and everything else in Twig files?

But no : since twig file came from QuestionTheme … if it's only in twig file : we can not add class to all  dependent theme …

The only reason is to try to have something clear for Theme dev … maybe we must put an information at top of Global classes for question part `deprecated in 6.X` …
This make harder to create global solution but … who really care …

Reply:
> > if it's only in twig file : we can not add class to all dependent theme …
> 
> Well, is that possible now? :)

Not always, but if QuestionTheme dev use {{ coreClass }} it's `ls-answers answers-list` currently for gender button here.

But maybe we just need to put https://manual.limesurvey.org/Theme_editor#Global_classes_for_question_part as global rule and hope QuestionTheme dev respect it …

Reply:
So maybe the important part is not that we keep the $coreClass variable, but that the manual (https://manual.limesurvey.org/Theme_editor#Global_classes_for_question_part) is correct and up to date and doesn't change more than it has to when moving to Bootstrap 5?

Reply:
> So maybe the important part is not that we keep the $coreClass variable, but that the manual (https://manual.limesurvey.org/Theme_editor#Global_classes_for_question_part) is correct and up to date and doesn't change more than it has to when moving to Bootstrap 5?

Yes, sure. But i think have some core part for QuetsionTheme dev make it easy.

Reply:
PS : i think with 6.X : we must review the JS action by class ?
https://github.com/LimeSurvey/LimeSurvey/blob/b4695ff55b91d214da64851036772467f5292614/assets/packages/expressions/em_javascript.js#L210

Maybe by data-em on input ?
Adding this class fix a lot of issue about part not updated when value is updated.

Reply:
yes that might be a better solution

Reply:
for javatbd : need to rework expression_manager … complex